PhysicsTools/PatAlgos/plugins/JetCorrFactorsProducer.cc

```diff
@@ -232,12 +232,9 @@ JetCorrFactorsProducer::produce(edm::Event& event, const edm::EventSetup& setup)
 					    << "the sequence.";
     }
     for(unsigned int idx=0; idx<corrLevel->second.size(); ++idx){
-      bool flavorDependent=false;
       std::vector<float> factors;
-      if(flavorDependent ||
-	 corrLevel->second[idx].find("L5Flavor")!=std::string::npos ||
+      if(corrLevel->second[idx].find("L5Flavor")!=std::string::npos ||
```

Comment:
@uhussain: I think we have our answer. LUE and L7Parton are obsolete and unused. We can go ahead with the current version of the file in the PR now, except delete the comment on line 238 since it is no longer relevant.
@slava77: Do you have any remaining concerns?